### DISCUSSION TITLE
Insert timestamp in the patterns

### DISCUSSION THREAD

**Discussion by e-alizadeh** [jglev/obsidian-apply-patterns-plugin#45] [Q&A] [answered]
First, thanks for the plugin. 

Is it possible to insert timestamp in the pattern. I'm using both QuickAdd and Templater and it would be nice to parse timestamps instead of literally replace the text. Basically I'm asking if it's possible to parse `{{DATE:YYYY-MM-DD}}` (QuickAdd syntax) or `<% tp.date.now()%>` (Templater syntax)

**Comment by jglev** [comment, marked as answer]
> First, thanks for the plugin.

You're welcome!

> Is it possible to insert timestamp in the pattern.

It is! Natural language dates are supported, using `{{date}}` or `{{date|two weeks from today}}` syntax. See [this section](https://github.com/jglev/obsidian-apply-patterns-plugin#dates) of the Readme for examples.

Is that responsive to your question?

**Reply by e-alizadeh** [reply]
Thanks for your response. It's working now. I had syntax issues as I was using `DATE` (following syntax from QuickAdd plugin) instead of `date`.

**Reply by jglev** [reply]
You're welcome! This perhaps points to a good place to update the plugin's documentation in the future, too.
